tests/kuttl/tests/run_simple_playbook/01-assert.yaml

```diff
@@ -72,12 +72,12 @@ commands:
           IMG_FROM_ENV=$(echo $ITEM | sed -e 's|^.*#\(.*\)|\1|')
           template='{{.spec.image}}'
           case $NAME in
-            API)
+            IMAGE)
```

Comment:
Forgive me, but that var `IMAGE` is set somewhere right?

Reply:
If I understood correctly what the script is meant to do, `IMAGE` is not a variable but a string literal